packages/ember/tests/routing/basic_test.js

```diff
@@ -89,6 +89,7 @@ QUnit.module('Basic Routing', {
 
       originalLoggerError = Logger.error;
       originalRenderSupport = ENV._ENABLE_RENDER_SUPPORT;
+      originalEnabledOrphanedOutletSupport = ENV._ENABLE_ORPHANED_OUTLETS_SUPPORT;
 
       ENV._ENABLE_RENDER_SUPPORT = true;
```

Comment:
Now I'm wondering if we should set `ENV._ENABLE_ORPHANED_OUTLETS_SUPPORT` to true like we do for `ENV._ENABLE_RENDER_SUPPORT` here?

Reply:
No,

By setting it to `true` we hit the old deprecation flow for all test which is what we are moving away from (and we want all of the tests to go through the new assertion flow). Currently we have just the test below set to `true` so only that one test verifies that the "old" path still works.